lib/v2/foreverblog-cn/feeds.js

```diff
@@ -0,0 +1,35 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const currentUrl = `https://www.foreverblog.cn/feeds.html`;
+
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
```

Comment:
```suggestion
    const response = await got(currentUrl);
```
Got request can be simplified